cloud/jenkins/pbm_docker_build.groovy

```diff
@@ -11,18 +11,18 @@ void build(String IMAGE_PREFIX){
         sudo rm -rf ./vendor
     """
 }
-void checkImageForDocker(String IMAGE_PREFIX){
+void checkImageForDocker(String IMAGE_SUFFIX){
      withCredentials([usernamePassword(credentialsId: 'hub.docker.com', passwordVariable: 'PASS', usernameVariable: 'USER')]) {
         sh """
-            IMAGE_PREFIX=${IMAGE_PREFIX}
+            IMAGE_SUFFIX=${IMAGE_SUFFIX}
             IMAGE_NAME='percona-server-mongodb-operator'
-            TrityHightLog="$WORKSPACE/trivy-hight-\$IMAGE_NAME-${IMAGE_PREFIX}.log"
-            TrityCriticaltLog="$WORKSPACE/trivy-critical-\$IMAGE_NAME-${IMAGE_PREFIX}.log"
+            TrityHightLog="$WORKSPACE/trivy-hight-\$IMAGE_NAME-${IMAGE_SUFFIX}.log"
+            TrityCriticaltLog="$WORKSPACE/trivy-critical-\$IMAGE_NAME-${IMAGE_SUFFIX}.log"
 
             sg docker -c "
                 docker login -u '${USER}' -p '${PASS}'
-                /usr/local/bin/trivy -o \$TrityHightLog --ignore-unfixed --exit-code 0 --severity HIGH --quiet --auto-refresh perconalab/\$IMAGE_NAME:main-${IMAGE_PREFIX}
-                /usr/local/bin/trivy -o \$TrityCriticaltLog --ignore-unfixed --exit-code 1 --severity CRITICAL --quiet --auto-refresh perconalab/\$IMAGE_NAME:main-${IMAGE_PREFIX}
+                /usr/local/bin/trivy -q --cache-dir /mnt/jenkins/trivy/ image -o \$TrityHightLog --timeout 10m0s --ignore-unfixed --exit-code 0 --severity HIGH perconalab/\$IMAGE_NAME:main-\${IMAGE_SUFFIX}
```

Comment:
I think it could create conflicts if pipeline was interrupted, so maybe better to use `--cache-dir /mnt/<name_of_pipeline>/${BUILD_NUMBER}`

Reply:
good catch!
I have reworked path, now `pbm_docker_build` job shouldn't affect `psmdb_docker_build` job.
it is impossible to run two `pbm_docker_build` jobs simultaneously because `disableConcurrentBuilds()` option is in place.